### ISSUE TITLE
Network Error when opening a closed project

### ISSUE THREAD

**Issue by kerem0comert** [go-gitea/gitea#30901]
### Description

Hi,

I have a closed project that I want to open - however when I click on the ```Open``` button I get a ```Network Error``` message as seen in the screenshot and the project does not open. When I look at the error logs from the browser console I see:

```
error when doRequest SyntaxError: Unexpected token '<', "<!DOCTYPE "... is not valid JSON
```

Any ideas why this could be the case?

### Gitea Version

1.21.5

### Can you reproduce the bug on the Gitea demo site?

No

### Screenshots

![networkerror](https://github.com/go-gitea/gitea/assets/42437138/ac758f95-c6e1-4d96-83cc-340b5ea3a449)

### Operating System

Linux

### How are you running Gitea?

Using one of the downloads

### Database

None

**Comment by wxiaoguang** [comment]
I did a quick test with docker, I can not reproduce the problem.

Do you see any related error logs in gitea's log and reverse proxy's log?

```
version: "3"

services:
  server:
    image: gitea/gitea:1.21.5
    container_name: gitea-1.21
    volumes:
      - ./gitea:/data
    ports:
      - "3001:3000"
      - "3021:22"
```

**Comment by yp05327** [comment]
I think this is fixed by #30387

**Comment by wxiaoguang** [comment]
> only org/individual project will got this error.

Good catch, it is.
